samples/getting-started/android/build.gradle

```diff
@@ -4,6 +4,7 @@ buildscript {
     ext.kotlin_version = '1.5.31'
     repositories {
         google()
+        jcenter()
```

Comment:
I removed this `jcenter()` in [this PR](https://github.com/flowkey/UIKit-cross-platform/pull/340/files#diff-2e072c2476eea21c7c03216f034ba0827afea76c24e55b05f2f37693592b7f9bL7) a while ago. What dependency are does depend on it and can we instead use maven?

Reply:
fair point, It was failing without jcenter but only because that specific version of the build tools didn't exist in the maven or google repo.

this commit fixes the build without jcenter()
https://github.com/flowkey/UIKit-cross-platform/pull/347/commits/d4234be2b9ebefaa59f478ce99a376de2d753c88